examples/egl_common.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _EGL_COMMON_H
+#define _EGL_COMMON_H
+#endif
+
+#include <stdbool.h>
+#include <wayland-client.h>
+
+#include <EGL/egl.h>
+#include <EGL/eglext.h>
+
+struct egl_ctx {
```

Comment:
Can we just get rid of this struct altogether? There will always ever be a single context for each example.

```c
extern EGLDisplay egl_display;
extern EGLConfig egl_config;
extern EGLContext egl_context;

bool init_egl(struct wl_display *display);
```

Reply:
Done, I think.